python/pyspark/heapq3.py

```diff
@@ -865,7 +865,7 @@ def nlargest(n, iterable, key=None):
 
 # If available, use C implementation
 try:
-    from _heapq import *
+    from _heapq import *  # noqa: F401
```

Comment:
This too. It was ported back from CPython 3, and is excluded in other linter checks. Can we exclude this file as well?

Reply:
Yes, let us not touch these files.